.travis.yml

```diff
@@ -0,0 +1,22 @@
+dist: bionic
+git:
+  submodules: false
+before_install:
+  - git clone https://${TOKEN}:x-oauth-basic@github.com/meedan/configurator ./configurator
+before_script:
+  - sed -i 's/git@github.com:/https:\/\/github.com\//' .gitmodules
+  - git submodule update --init --recursive
+  - sed -i 's/--abort-on-container-exit/-d/' bin/first-build.sh
+  - cd fetch
+  - git checkout develop
+  - TAB=$'\t'
+  - sed -i '/-c 5/ a \'"${TAB}"'\ until curl --silent -XGET --fail http://elasticsearch:9200; do printf \'.'\; sleep 1; done' Makefile
+  - cd -
+script:
+  - ./bin/first-build.sh
+  - docker ps --format '{{.Image}}' | sort >> docker_up_output.txt
+  - DIFF=$(diff docker_up_output.txt test/image_names.txt)
+  - if [ "${DIFF}" != "" ];  then exit 1; fi;
```

Comment:
In order to be testing even more here, I suggest to run a few `curl` commands to be sure that the services are running properly... `curl -I -f <URL>` will return an exit code greater than zero if the answer is not 200, so it's enough to just call them... if any of the calls returns something different than 200, the build should fail (but please be sure it does). We only need to test the most important services:

```
- curl -I -f http://localhost:3100
- curl -I -f http://localhost:3000
- curl -I -f http://localhost:3333
- curl -I -f http://localhost:3200
```